tmt/steps/provision/vagrant.py

```diff
@@ -111,27 +277,34 @@ def destroy(self):
             except GeneralError:
                 sleep(5)
 
+    def ansible(self, playbook):
+        """ Prepare guest using ansible playbook """
+        self.prepare("ansible", playbook)
+
     def prepare(self, how, what):
-        """ add single 'preparator' and run it """
+        """ Add single 'preparation' and run it.
+        """
 
-        name = 'prepare'
+        self.preparations += 1
+        name = 'prepare_' + self.preparations
         cmd = 'provision'
 
         self.vf_backup("Prepare")
 
         # decide what to do
         if how == 'ansible':
-            name = how
+            what = self._ansible_playbook_path(what)
 
             # Prepare verbose level based on the --debug option count
             verbose = self.opt('debug') * 'v' if self.opt('debug') else 'false'
+
             self.add_config_block(cmd,
                 name,
                 f'become = true',
-                self.kve('become_user', self.data['user']),
+                self.kve('become_user', self.user),
```

Comment:
Could we please have a more clear name for `kve`?

Reply:
"Key Value Equals" .. It's a simple formatting helper. Any suggestions?

Reply:
Don't know. Perhaps `format_key_value` or `key_equals_value`?

Reply:
Seems rather unnecessary long for a `'{key}{sep}{quote(val)}'` ... but If it ehnances readability ...

What about `format_equals`?

Reply:
Sounds good/better.